### ISSUE TITLE
Master: mono zigate -> mutli zigate: Call to a member function getHumanName() on boolean

### ISSUE THREAD

**Issue by edgd1er** [KiwiHC16/Abeille#984]
URL page: index.php?v=d&m=Abeille&p=Abeille

For an unknown reason, with the latest master, I have this error. If I remove "->getHumanName()", then the Abeille plugin page can be loaded. I suspect the function Abeille::byLogicalId( 'Abeille'.$i.'/Ruche', 'Abeille') to return a false value.

The code is looking for a Abeille1/Ruche in logicalId, but in the Database, I have only a Abeille/Ruche field content. Hence the error.

Temporary can be either: 
* rename in the database the logicalId for the Ruche

```
UPDATE `jeedom`.`eqLogic` SET `logicalId`='Abeille1/Ruche' WHERE  `name`='Ruche';
```

* or remove ->getHumanName() in desktop/php/Abeille.php: L83

```
echo '<div id="bt_include'.$i.'" > Zigate'.$i . ' - ' . Abeille::byLogicalId( 'Abeille'.$i.'/Ruche', 'Abeille')->getHumanName() . ' <i class="fa fa-plus-circle" style="font-size:160%;color:green" title="Inclusion: clic sur le plus pour mettre la zigate en inclusion."></i> </div>';
```

I guess the best fix would be to update database, @KiwiHC16 may confirm that before users update their Abeille's config ;)

* What I have tried is to add the following code that would be "compatible" with legacy naming and new naming.

 **desktop/php/Abeille.php: L83**

```
 if (Abeille::getParameters()['AbeilleSerialPort' . $i] != "none") {
//// added -- start //////
$currentZigate = Abeille::byLogicalId('Abeille' . $i . '/Ruche', 'Abeille');
if ($i == 1 && $currentZigate == "") {
       $currentZigate = Abeille::byLogicalId('Abeille/Ruche', 'Abeille');
       }
//// added -- end //////
      echo '<div id="bt_include' . $i . '" > Zigate' . $i . ' - ' . $currentZigate->getHumanName() . ' <i class="fa fa-plus-circle" style="font-size:160%;color:green" title="Inclusion: clic sur le plus pour mettre la zigate en inclusion."></i> </div>';
```

**Comment by KiwiHC16** [comment]
```
The code is looking for a Abeille1/Ruche in logicalId, but in the Database, I have only a Abeille/Ruche field content. Hence the error.
```

Oui il faut passer les "Abeille/xxxx" en "Abeille1/xxxx". La fonction doit être ajouté. Pas encore finalisé. J'ai un draft en cours.

C'est une des raisons qui fait que je n ai pas encore publié une stable multizigate.

**Comment by KiwiHC16** [comment]
Should be solved with #974 related push.
